### ISSUE TITLE
wired-input inside wired-dialog is invisible in react

### ISSUE THREAD

**Issue by niaodan2b** [rough-stuff/wired-elements#114]
```
import React from 'react';
import "wired-elements";

function App() {
  return (
    <div className="App">
      <wired-dialog open>
        <wired-input></wired-input>
      </wired-dialog>
    </div>
  );
}

export default App;
```
![](https://i.imgur.com/vwVYjVl.png)

**Comment by niaodan2b** [comment]
If I replace wired-input with wired-button, it's rendered correctly. Am i doing something wrong?

**Comment by pshihn** [comment]
Sorry for not looking into this sooner. Let me check

**Comment by notVitaliy** [comment]
Any updates on this? I'm having the same issue.

**Comment by pshihn** [comment]
This is fixed in v2.1.2 Here's ad demo in React: https://codesandbox.io/s/wired-elements-react-dialog-czckx?file=/src/index.js

**Comment by lizy0329** [comment]
> This is fixed in v2.1.2 Here's ad demo in React: https://codesandbox.io/s/wired-elements-react-dialog-czckx?file=/src/index.js

same question in "wired-elements": "^3.0.0-rc.6",
vite: 2.4.4
vue: 3.2.2

**Comment by lizy0329** [comment]
https://codesandbox.io/s/wired-elements-react-dialog-forked-21w40?file=/src/index.js

![企业微信截图_20210825155931.png](https://i.loli.net/2021/08/25/xYQBpRCIAzb2Lch.png)
